apps/fabric-website/src/pages/Overviews/ResourcesPage/docs/default/ResourcesDeveloperResources.md

```diff
@@ -7,6 +7,15 @@ Get started with React and learn how to build your first projects.
   <li class="mdut--full">[Fabric React tutorial](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Getting-Started-with-UI-Fabric)</li>
 </ul>
 
+### Documentation
+
+Find detailed documentation of the API along with examples of each control on the Fabric website or a full collection of the API references on the docs.microsoft.com portal.
```

Comment:
Since this page is primarily for links to external resources, I think something like the following would be better (then remove the "Fabric website controls page" link from the list below):
```suggestion
In addition to the API documentation and control examples found in the [Controls section](#/controls/web) of this website, you can find a full collection of API references on the docs.microsoft.com portal.
```

Reply:
I'm also not sure that "a full collection of API references" is quite the right wording--possibly "a full API reference" or "full API references" would be better.